core/src/services/hdfs/backend.rs

```diff
@@ -85,6 +99,12 @@ impl HdfsBuilder {
         }
         self
     }
+
+    /// Set enable_append of this backend
```

Comment:
Please add comments on why we need this and how to enable append support on hdfs.

Reply:
done.